app/src/main/java/com/github/se/icebreakrr/ui/sections/AroundYou.kt

```diff
@@ -68,9 +68,7 @@ fun AroundYouScreen(
                 items(profiles.value.size) { index ->
                   ProfileCard(
                       profile = profiles.value[index],
-                      onclick = {
-                        Toast.makeText(context, "Profile clicked", Toast.LENGTH_SHORT).show()
-                      })
+                      onclick = { navigationActions.navigateTo("Settings Screen") })
```

Comment:
question: Why `"Settings Screen"` instead of a route like `"Profile View"` ? But most importantly, why does it work ???

Reply:
Because I place the ProfileView inside the settings screen temporarily